CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,3 @@
+## Code of Conduct
+
+Flux and its sub-projects follow the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
```

Comment:
We went to great lengths to avoid "sub projects", maybe just copy the file from `fluxcd/community`?

Reply:
Done!